bin/akula-ddl.rs

```diff
@@ -0,0 +1,34 @@
+use akula::downloader::{
+    chain_config, chain_fork_config::ChainForkConfig, chain_id::ChainId, opts::Opts, sentry_client,
+    sentry_client::SentryClient,
+};
+
+use std::iter::FromIterator;
+use tokio;
+use tracing_subscriber::EnvFilter;
+
+async fn run() -> anyhow::Result<()> {
+    let chains_config = chain_config::ChainsConfig::new()?;
+    let chain_names = Vec::from_iter(chains_config.0.keys().map(|k| k.as_str()));
+    let opts = Opts::new(chain_names.as_slice())?;
+    let chain_config = &chains_config.0[&opts.chain_name];
+    let status = sentry_client::Status {
+        chain_id: ChainId::from_config(chain_config)?,
+        total_difficulty: ethereum_types::H256::from([0 as u8; 32]),
```

Comment:
it's a number, use `ethereum_types::U256` instead

Reply:
:+1: